payload_designer/pipelines/grism_eff_vs_fringe_density.py

```diff
@@ -0,0 +1,108 @@
+"""Calculate grism efficiency for VPH grism from fringe density, incidence
+angle, order of diffraction, grating volume thickness, refractive index of VPH
+and its semiamplitude."""
```

Comment:
I might not have made myself very clear on this, but we aren't actually creating pipeline scripts of this sort anymore. We've moved to using interactive Jupyter notebooks. This folder is still around because not all of these pipeline scripts have been fully migrated to the new framework. Let's rename the `pipelines` folder to `pipelines_LEGACY` to make it clear we aren't creating these non-Jupyter notebook scripts anymore. You should keep only the notebook script you wrote to produce the graphs